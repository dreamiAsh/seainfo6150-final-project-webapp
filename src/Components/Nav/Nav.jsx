import React from "react";
import styles from "./Nav.module.css";
import { Link } from "react-router-dom";

const Nav = (props) => {
  return (
          <ul className={styles.box}>
            {/* these links should show you how to connect up a link to a specific route */}
            <li className={styles.liCover}>
              <Link to="/" className={styles.li}>Home</Link>
            </li>
            <li className={styles.liCover}>
              <Link to="/news" className={styles.li}>News</Link>
            </li>
            <li className={styles.liCover}>
              <Link to="/filter" className={styles.li}>Filter</Link>
            </li>
            <li className={styles.liCover}>
              <Link to="/aboutUs" className={styles.li}>About us</Link>
            </li>
          </ul>
  );
};

export default Nav;
