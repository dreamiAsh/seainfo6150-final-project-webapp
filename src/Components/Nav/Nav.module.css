.box{
    width: 100%;
    height: 30px;
    background: #E5B687;
    display: grid;
    flex-flow: column-reverse;
    justify-content: space-around;
    align-items: stretch;
    grid-template-columns: repeat(4, 30fr);
    text-align: center;
}
.li{
    line-height: 30px;
    font-size: 15px;
    font-style: normal;
    color: white;
    text-decoration: none;
    font-weight: bold;
}
.liCover:hover{
    height: 30px;
    background: #D05518;
}