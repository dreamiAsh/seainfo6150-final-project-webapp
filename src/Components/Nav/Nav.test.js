import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Nav from "./Nav.jsx";

it("renders the Nav component correctly", () => {
  const { container } = render(<Nav />);
  expect(container).toMatchSnapshot();
});