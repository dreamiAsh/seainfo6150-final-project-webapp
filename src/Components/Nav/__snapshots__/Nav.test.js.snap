// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Nav component correctly 1`] = `
<div>
  <ul
    class="box"
  >
    <li
      class="liCover"
    >
      <a
        class="li"
        href="/"
      >
        Home
      </a>
    </li>
    <li
      class="liCover"
    >
      <a
        class="li"
        href="/news"
      >
        News
      </a>
    </li>
    <li
      class="liCover"
    >
      <a
        class="li"
        href="/filter"
      >
        Filter
      </a>
    </li>
    <li
      class="liCover"
    >
      <a
        class="li"
        href="/aboutUs"
      >
        About us
      </a>
    </li>
  </ul>
</div>
`;
